src/App.js

```diff
@@ -7,6 +7,8 @@ import {VideoDetails} from './pages/VideoDetails/VideoDetails';
 import {Like} from './pages/Like/Like';
 import {WatchLater} from './pages/WatchLater/WatchLater'
 import {History} from './pages/History/History'
+import {Playlist} from './pages/Playlist/Playlist'
+import { PlaylistDetails } from './pages/PlaylistDetails/PlaylistDetails';
```

Comment:
Just a suggestion,  create ``index.js`` file and then export all the pages you need in your app from there

Reply:
i did that may be i forgot here

Reply:
login with test is also working fine